examples/plot_alignment_methods_benchmark.py

```diff
@@ -163,11 +145,109 @@
         cut_coords=[-15, -5],
         vmax=1,
         title=title,
+        axes=axes[i],
+        colorbar=False,
     )
 
 ###############################################################################
-# We can observe that all alignment methods perform better than identity
-# (no alignment). Ridge is the best performing method, followed by Optimal
-# Transport. If you use Ridge though, be careful about the smooth predictions
-# it yields.
+# Next, let's compare the template-based alignment methods to the Shared
+# Response Model (SRM) method. SRM computes a shared response space from
+# different subjects to a particular task, and then projects individual subject
+# data into this shared space.
+
+###############################################################################
+# Initialize the IdentifiableFastSRM model. This version of SRM ensures that
+# the solution is unique.
+#
+
+from fastsrm.identifiable_srm import IdentifiableFastSRM
+
+srm = IdentifiableFastSRM(
+    n_components=30,
+    n_iter=10,
+)
+
+# #############################################################################
+# For the SRM method, we will use the same training data as before
+# (source subjects: sub-01 and sub-02) and the target subject (sub-04), and we
+# will divide our source and target subjects into training and testing sets,
+# leveraging the AP and PA acquisitions.
+
+# The training set:
+# * source train: AP acquisitions from source subjects (sub-01, sub-02).
+#   These will be projected into a shared response space.
+# * target train: AP acquisitions from the target subject (sub-04).
+#   These define the target shared space.
 #
+
+source_subjects = [sub for sub in sub_ids if sub != "sub-04"]
+source_train = [
+    concat_imgs(df[(df.subject == sub) & (df.acquisition == "ap")].path.values)
+    for sub in source_subjects
+]
+target_train = concat_imgs(
+    df[(df.subject == "sub-04") & (df.acquisition == "ap")].path.values
+)
+
+# The testing set:
+# * source test: acquisitions from source subjects (sub-01, sub-02).
+#   These will be projected into the shared space and transformed to predict
+#   the target.
+# * target test: PA acquisitions from the target subject (sub-04).
+#   These serve as ground truth to evaluate prediction accuracy.
+#
+
+source_test = [
+    concat_imgs(df[(df.subject == sub) & (df.acquisition == "pa")].path.values)
+    for sub in source_subjects
+]
+target_test = concat_imgs(
+    df[(df.subject == "sub-04") & (df.acquisition == "pa")].path.values
+)
+
```

Comment:
Similarly, this can also be removed in favor of L90 !

Reply:
In L90 we're not splitting the "ap" and "pa" acquisitions... I can modify this lines so the name matches? or I can do the split since the beginning and just concatenate all of them for the template method

Reply:
Thanks for confirming ! 

I definitely see the confusion, and you're right that for most TemplateAlignment methods we could combine the AP-PA acquisitions, but for fairness I think that have to leave out `source_test` (otherwise two of the methods will learn on more data, so they should innately do better than SRM). It would just make the results harder to compare !